tests/ioctl/test_ioctl.c

```diff
@@ -28,9 +28,9 @@ int main(int argc, char **argv)
 	}
 
 	/* This one should hit the FILE__GETATTR or FILE__IOCTL test */
-	rc = ioctl(fd, FIGETBSZ, &val);
+	rc = ioctl(fd, FS_IOC_GETFLAGS, &val);
```

Comment:
Haven't yet had a chance to go through all the changes, but this part seems to make the `ioctl` test fail under NFS (see `tools/nfs.sh`). Can you look into it?